endoreg_db/views/patient_finding_views.py

```
@@ -0,0 +1,69 @@
+        # Additional logic for creating patient findings
+        patient_finding = serializer.save()
+        return patient_finding
+
+@api_view(['POST'])
+def create_patient_finding_location(request):
+    """
+    Create a PatientFindingLocation entry
+    """
+    serializer = PatientFindingLocationSerializer(data=request.data)
+    if serializer.is_valid():
+        location = serializer.save()
```

Ruff (0.11.9)

Local variable `location` is assigned to but never used

Remove assignment to unused variable `location`

(F841)